src/components/drop/DropPod.tsx

```diff
@@ -134,6 +153,12 @@ class RDropPod extends React.PureComponent<IRDropPodProps, IDropPodState> {
     }
 
     render() {
+        if (this.props.isOpen) {
+            this.setEventsOnDocument();
+        } else {
+            this.removeEventsOnDocument();
+        }

```

Comment:
Same here. This if block should be done inside `componentDidUpdate`, because you will be setting a new event handler on each render if `this.props.isOpen` remains true.